plugins/processors/ec2tagger/ec2tagger.go

```diff
@@ -309,25 +242,22 @@ func (t *Tagger) Init() error {
 			t.metadataLookup.instanceId = true
 		case mdKeyImageId:
 			t.metadataLookup.imageId = true
-		case mdKeyInstaneType:
+		case mdKeyInstanceType:
 			t.metadataLookup.instanceType = true
 		default:
-			t.Log.Errorf("ec2tagger: Unsupported EC2 Metadata key: %s", tag)
+			t.Log.Errorf("ec2tagger: Unsupported EC2 Metadata key: %s.", tag)
 		}
 	}
 
-	t.Log.Infof("ec2tagger: Check ec2 metadata")
-	if !t.ec2metadata.Available() {
-		msg := "ec2tagger: Unable to retrieve InstanceId. This plugin must only be used on an EC2 instance"
-		t.Log.Errorf(msg)
-		return errors.New(msg)
-	}
+	t.Log.Infof("ec2tagger: EC2 IMDS has started initialization.")
```

Comment:
Nit:
1. This doesn't need to be `Infof()` because there is no format string used.
2. The log itself is confusing. It is not immediately obvious what "has started initialization" means in this context. This could be interpreted as the IMDS service is starting on the instance. 

What value does this log provide over the existing one?

Reply:
No deep reason behind it. I forgot to change back when merging from the telegraf upgrade. Nice catch on this 👍